### ISSUE TITLE
Icon request: icon-naira

### ISSUE THREAD

**Issue by aniebiet** [FortAwesome/Font-Awesome#1969]
- For indicating services that cost money to a Nigerian 

![naira_sign](https://f.cloud.github.com/assets/2031040/1144218/df56eef2-1d7c-11e3-9aac-0ed1462b6f28.gif)

**Comment by bosunolanrewaju** [comment]
+1

**Comment by Sammyskills** [comment]
Please include this, it will relieve a lot of stress.

**Comment by sdkcodes** [comment]
I'd like to see the naira icon added to font-awesome as well.

**Comment by texyh** [comment]
cool, using it already

**Comment by mister-tobbhie** [comment]
@texyh evening bro, but I dont see d css code anywhere here, so please where did u see urs

**Comment by timothymax** [comment]
don't know whether it has been added yet. but it will really be useful

**Comment by nasiriyima** [comment]
I recently downloaded the latest version of fontawesome, v5.0.13 as at the time of writing this post in 2018, but I couldn't find any css code matching the Naira Symbol. Hence,I  did a little work around by adding the Naira symbol css code `\0020A6` in fontawesome stylesheet file and I called it `fa-ngn.` Though I might be wrong but I couldn't find any reference to the Naira symbol in the fontawesome stylesheet. Is there any ongoing work on this issue or has it been resolved in a version I'm missing?
